.github/workflows/release.yml

```diff
@@ -86,9 +91,35 @@ jobs:
     steps:
       - run: echo "Unit test finished"
 
-  #Once all the tests have finished, we publish every module in parallel
-  publish_core:
+  #Once all the tests have finished, we update version and tag it
+  change_version:
     needs: [ unit_test_finished ]
+    name: Change Version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set github user
+          # We set these credentials because we have no way of getting the users (that started the workflow) email
+        # Reference: https://github.com/actions/checkout/issues/13
+        run: |
+          git config user.name github-actions[bot]
```

Comment:
Should we maybe put this as blocked until we can get an actual account for this? Also, I think we still need to update the Sonatype user in the secrets here to a group account. I don't think we'd want to publish using this script until both of those are complete.

Reply:
@sarahkoop the sonatype account is updated! its the user `braintree` in 1Password under Sonatype for Android publishing. Or do you mean creating a new one that its not `braintree`? perhaps a paypal one?

Reply:
Ah I missed that ok great!